packages/@react-aria/overlays/test/useOverlay.test.js

```diff
@@ -40,6 +40,22 @@ describe('useOverlay', function () {
     expect(onClose).toHaveBeenCalledTimes(1);
   });
 
+  it('should not hide the overlay when clicking outside if shouldCloseOnInteractOutside returns true', function () {
```

Comment:
typo i think? `should hide`
not
`should not hide`

Reply:
oops! :sweat_smile: